client/src/Components/Header/Header.js

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment } from "react";
+import "./Header.scss";
+import logo from "../../../public/static/images/logo.png";
+import CartIcon from "../CartIcon/CartIcon";
+
+const Header = () => {
+  return (
+    <header className="header">
+      <img src={logo} alt={"Sabka Bazaar Logo"} height="60" width="140" className="logo" />
```

Comment:
no need to give specific class to header. You can directly access header in css

Reply:
fixed it.